test/tests/outputs/csv/tests

```diff
@@ -68,6 +68,14 @@
     []
   []
 
+  [all_columns]
+    type = CSVDiff
+    input = 'all_columns_parent.i'
+    csvdiff = 'all_columns_parent_out.csv'
+    requirement = "The system shall allow new CSV rows even if times are the same, as long as some column is different."
```

Comment:
umm the wording seems too casual?

Reply:
how about "The system shall allow multiple lines to be output in a CSV file with the same physical time if those lines are not duplicates."